op-chain-ops/script/cheatcodes_utilities.go

```diff
@@ -188,5 +191,77 @@ func (c *CheatCodesPrecompile) Breakpoint_f7d39a8d(name string, v bool) {
 	}
 }
 
+// ParseTomlAddress_65e7c844 implements https://book.getfoundry.sh/cheatcodes/parse-toml. This
+// method is not well optimized or implemented. It's optimized for quickly delivering OPCM. We
+// can come back and clean it up more later.
+func (c *CheatCodesPrecompile) ParseTomlAddress_65e7c844(tomlStr string, key string) (common.Address, error) {
```

Comment:
Why the `_65e7c844` suffix?

Reply:
Go doesn't support overloads, so the suffix is the 4byte selector for this specific overload.